.drone.yml

```diff
@@ -75,50 +131,19 @@ steps:
     - refs/tags/*
     event:
     - tag
-
 volumes:
 - name: docker
   host:
     path: /var/run/docker.sock
-
+depends_on:
```

Comment:
same question above...If you combined build in one step you probably don't need `depend_on`

Reply:
Agreed